client/src/components/common/Button/index.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import PropTypes from "prop-types";
+import useStyles from "./style";
+
+const CustomButton = ({ children, customStyle }) => {
+  const classes = useStyles();
+  return <button className={classes[`${customStyle}`]}>{children}</button>;
+};
+
+CustomButton.propTypes = {
+  // eslint-disable-next-line react/forbid-prop-types
+  classes: PropTypes.object.isRequired,
+  children: PropTypes.string.isRequired,
+  customStyle: PropTypes.string,
+};
+CustomButton.defaultProps = {
+  customStyle: "contained", //or outlined
+  children: "next",
+};
```

Comment:
delete the children cause it Required prop type & the Required ones didn't take a default value.
```suggestion
CustomButton.defaultProps = {
  customStyle: "outlined", 
  disabled: false, 
   onClick: () => {},  
};
```